txstratum/prometheus.py

```diff
@@ -109,6 +88,62 @@ def _schedule_and_write_data(self) -> None:
             loop = asyncio.get_event_loop()
             loop.call_later(self.call_interval, self._schedule_and_write_data)
```

Comment:
This sequence of calls is not reliable. Any unhandled exception in `update_metrics` will prevent the next schedule and thus prometheus will stop updating since then. I feel we should move to LoopingCall.

Reply:
Done